gallery/src/styles/NxFormLayout/NxFormLayoutExample.tsx

```diff
@@ -90,6 +96,18 @@ export default function NxFormLayoutExample() {
           Blue
         </NxRadio>
       </fieldset>
+      <div className="nx-form-group">
+        <label className="nx-label">
+          <span className="nx-label__text">Select</span>
+          <select className="nx-form-select" value={selectVal} onChange={onSelectChange}>
```

Comment:
Notice that when the page first loads, the `selectVal` variable holds `''` but the select is set to "Option 1".  So at that point is time the state variable and the actual UI are out of sync which is undesirable.  To avoid this, set the default value of the state variable (that is, the value passed into the `useState` call), to a value that is actually one of the valid choices from the `<select>`

Reply:
> Notice that when the page first loads, the `selectVal` variable holds `''` but the select is set to "Option 1". So at that point is time the state variable and the actual UI are out of sync which is undesirable. To avoid this, set the default value of the state variable (that is, the value passed into the `useState` call), to a value that is actually one of the valid choices from the `<select>`

What I should have done is have a "Pick one" option specified first: https://github.com/sonatype/sonatype-react-shared-components/pull/203/commits/a329a17b7cda35bbf29e06fca284f1b3ae6255ea